admin-front/src/components/Nfts.tsx

```diff
@@ -167,13 +200,46 @@ const NftAside = ({ ...props }) => {
                         {categories.map(category => {
                           if (!category) return;
                           return category.name + (categories.indexOf(category) === categories.length - 1 ? '' : ', ')
-                        }
-                        )
+                        })
                         }
                       </Typography>
                     </Stack>
                   </Stack>
                 )
+                if (attrKey === "proposal_vote" && voters.length > 0) {
```

Comment:
Shouldn't map on field identifiers here, rather on the kinds they are. If you don't have the relevant info coming from BE than we should quickly chat about that.